docs/traffic-policy/actions/add-headers/config.mdx

```diff
@@ -0,0 +1,23 @@
+## Configuration Reference
+
+This is the [Traffic Policy](/docs/http/traffic-policy/index.mdx) configuration 
+reference for this action.
+
+### Action Type
+
+`add-headers`
+
+### Configuration Fields
+
+| Parameter | Type                    | Description                                                                                                                                              |
+| --------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `headers` | `map[string]string` | Map of header key to header value to be added. Minimum `1`, Maximum `10`. Supports [string interpolation](#string-interpolation) in header values. |
+
+### Supported Directions
+
+- `inbound`
+- `outbound`
+
+### Supported Schemes
```

Comment:
nit: under http, are there any actions for which only one or none of these schemes is valid?  if not maybe can remove as not an important distinction.  or if it's important to include since we think it helps users understand, i think should add to http actions (eg log?) that are missing this section

Reply:
Kind of preparing for future content with this.